src/highdicom/pr/sop.py

```diff
@@ -145,63 +157,45 @@ def __init__(
         modality_lut: Union[highdicom.ModalityLUT, None], optional
             Lookup table specifying a pixel rescaling operation to apply to
             the stored values to give modality values.
-        copy_modality_lut: bool, optional
-            Include elements of the Modality LUT module (including
-            RescaleIntercept, RescaleSlope and ModalityLUTSequence), if any,
-            in the presentation state with values copied from the source images.
         softcopy_voi_luts: Union[Sequence[highdicom.pr.SoftcopyVOILUT], None], optional
             One or more pixel value-of-interest operations to be applied after
             the modality LUT and/or rescale operation. Note that multiple
             items should only be provided if no image, or frame within a
             multi-frame image, is referenced by more than one item.
-        copy_voi_lut: bool, optional
-            Include elements of the Softcopy VOI LUT module (including
-            WindowWidth, WindowCenter, and VOILUTSequence), if any, in the
-            presentation state with values copied from the source images.
-        icc_profile: Union[bytes, None], optional
-            ICC color profile to include in the presentation state. If none is
-            provided, the profile will be copied from the referenced images.
-            The profile must follow the constraints listed in :dcm:`C.11.15
-            <part03/sect_C.11.15.html>`.
+        presentation_lut_shape: Union[highdicom.pr.PresentationLUTShapeValues, str, None], optional
+            Shape of the presentation LUT transform, applied after the softcopy
+            VOI transform to create display values.
+        presentation_luts: Optional[Sequence[highdicom.LUT]], optional
+            LUTs for the presentation LUT transform, applied after the softcopy
+            VOI transform to create display values.
         transfer_syntax_uid: Union[str, highdicom.UID], optional
             Transfer syntax UID of the presentation state.
         **kwargs: Any, optional
             Additional keyword arguments that will be passed to the constructor
             of `highdicom.base.SOPClass`
 
         """  # noqa: E501
-        # Check referenced images are from the same series and have the same
-        # size
-        ref_im = referenced_images[0]
-        ref_series_uid = ref_im.SeriesInstanceUID
-        ref_im_rows = ref_im.Rows
-        ref_im_columns = ref_im.Columns
-        for im in referenced_images:
-            series_uid = im.SeriesInstanceUID
-            if series_uid != ref_series_uid:
-                raise ValueError(
-                    'All referenced images must belong to the same series.'
+        for kw in [
+            'icc_profile',
+        ]:
+            if kw in kwargs:
+                raise TypeError(
+                    'GrayscaleSoftcopyPresentationState() got an unexpected '
+                    f'keyword argument "{kw}".'
                 )
```

Comment:
This test is no longer needed if the class does not inherit from the pr base class.

Reply:
Previously it was possible for people to sneak through some of the keyword args for the base class that are not used for the subclass in the `**kwargs` of the subclass's constructor call so we had to guard against this. Yeah... ugly....

Reply:
Removed with bebf627a19aefebf08c0dcfb9c9058cefb2c2204